src/app/data-entries/select/select-header-footer/select-header-footer.component.scss

```
@@ -73,11 +73,11 @@ $custom-button-group: igx-button-group-theme(
     $schema: $light-schema,
     $shadow: none,
     $border-radius: 0,
-    $item-background: (igx-color: ('grays', 300)),
-    $item-text-color: (igx-color: ('grays', 700)),
-    $item-hover-background: (igx-color: ('grays', 400)),
-    $item-hover-text-color: (igx-color: ('grays', 700)),
-    $item-border-color: (igx-color: ('grays', 700)),
+    $item-background: igx-color('grays', 300),
+    $item-text-color: igx-color('grays', 700),
+    $item-hover-background: igx-color('grays', 400),
+    $item-hover-text-color: igx-color('grays', 700),
+    $item-border-color: igx-color('grays', 700),
```

This is wrong, it should be:

```suggestion
    $item-background: igx-color($color: 'grays', $variant: 300),
    $item-text-color: igx-color($color: 'grays', $variant: 700),
    $item-hover-background: igx-color($color: 'grays', $variant: 400),
    $item-hover-text-color: igx-color($color: 'grays', $variant: 700),
    $item-border-color: igx-color($color: 'grays', $variant: 700),
```